apiary.apib

```diff
@@ -2232,6 +2255,10 @@ Get a list of available tax rates.
                     + type: `department` (string)
                     + id: `182af0a8-3f68-409b-8941-cf8caf8f28a0` (string)
 
+## Payment Terms [/paymentTerms]
+
+*Required scopes: `invoices`*
```

Comment:
I think this should be in CRM... because you can set a paymentTerm per customer without having invoicing

Reply:
And what would you do with the paymentTerm besides create invoices?
I think the concept of a paymentTerm is only relevant for the invoicing module, so it's pointless to use it without the invoices scope.

Reply:
We can also drop this scope.